src/main/java/pl/wip/wipplatformbe/models/User.java

```diff
@@ -0,0 +1,50 @@
+package pl.wip.wipplatformbe.models;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import pl.wip.wipplatformbe.enums.Badge;
+import pl.wip.wipplatformbe.enums.Role;
+import pl.wip.wipplatformbe.enums.UserStatus;
+//import org.springframework.security:spring-security-crypto
+//import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+public class User {
```

Comment:
Class that represents a mongo document should have a Document annotation 
[Read this](https://www.mongodb.com/compatibility/spring-boot)